integration-tests/pages/createReport/evidencePage.js

```diff
@@ -1,5 +1,5 @@
-const page = require('../page')
-const CheckAnswersPage = require('./checkAnswersPage')
+import page from '../page'
+import CheckAnswersPage from './checkAnswersPage'
```

Comment:
something weird has happened with circle module system, the previous mismatch of module syntax has stopped working and now only the opposite mismatch will work